### ISSUE TITLE
Improve accuracy: more data, fine-tuning

### ISSUE THREAD

**Issue by rayen-feb** [rayen-feb/Tunisian-Sign-Language-recognition#3]
UNFREEZE STRATEGY
LEARNING RATE IMPROVEMEnt
Add more augmentation 
INCREASE INPUT QUALITY
BATCH SIZE TUNING
INCREASE INPUT QUALITY
EARLY STOPPING IMPROVEMENT
